packages/react-charting/src/utilities/ChartHoverCard/ChartHoverCardRTL.test.tsx

```diff
@@ -0,0 +1,105 @@
+import { render, screen } from '@testing-library/react';
+import * as React from 'react';
+import { ChartHoverCard } from './ChartHoverCard';
+import { getByClass } from '../TestUtility.test';
+
+describe('ChartHoverCard', () => {
+  test('renders the chart hover card component', () => {
+    const { container } = render(<ChartHoverCard />);
+    const chartHoverCardElement = getByClass(container, /calloutContentRoot/);
+    expect(chartHoverCardElement).toBeDefined();
+  });
+
+  test('displays the correct XValue', () => {
+    const XValue = 'Sample XValue';
+    const { container } = render(<ChartHoverCard XValue={XValue} />);
+    const XValueElement = screen.getByText(XValue);
+    expect(XValueElement).toBeInTheDocument();
+    expect(getByClass(container, /calloutContentX/)).toBeDefined();
```

Comment:
you are not checking for the actual value here

Reply:
but while getting the element itself, screen.getByText(XValue); ensures the value is same

Reply:
we need a more targeted way to getByText only in the intended container and displayed content in the expected target html element.

Reply:
done